examples/xds-management-server/cmd/controller.go

```diff
@@ -0,0 +1,94 @@
+package main
```

Comment:
organisation thought:

Should this be under `/cmd/agones/agones.go` (since it's an agones provider)

Then it's inline with `/cmd/file/file.go` and consistent.

Reply:
Not exactly, file.go is a different binary because its more of a demo use case (e.g proxy specific filterchain doesn't make sense there). While the main binary isn't specific to agones or k8s as we add more providers they'll still be in the same binary only with different configs

Reply:
Ah got it - that totally makes sense 👍🏻 I think we should make that clear in a README though, just so people understand which piece is which.